test/external/handle-filter

```diff
@@ -6,15 +6,15 @@ class Filter < Sensu::Handler
     puts "Event: #{@event.inspect}"
   end
 
-  def api_request(*)
+  def api_request(*_args)
```

Comment:
don't think that the args should be optional, if no args are given it still returns true which I doubt is expected.